processor/transform/gemoji.ts

```diff
@@ -0,0 +1,58 @@
+import { emoji as Emoji } from '@readme/emojis';
+import { Root } from 'mdast';
+import { nameToEmoji } from 'gemoji';
```

Comment:
@Dashron with the new `moduleResolution: 'bunder'`, `tsc` is getting confused when looking up this module. It looks like it's resolving to the same file??

![image](https://github.com/readmeio/markdown/assets/451488/5391d505-d9e9-4015-8fb0-75cb8bdc0750)

Reply:
i've had issues with this in the past and haven't been able to find a way around it short of renaming the file i see this in to something else.

Reply:
Yea ok. I tried that but got confused cause I have other issues too.